docs/3-building-blocks-of-oop-part-2/hometask/src/Sword.ts

```diff
@@ -1 +1,27 @@
-// your code goes here
\ No newline at end of file
+import { Weapon } from "./Weapon";
+
+const MAXIMUM_DAMAGE_MODIFIER_PERCENTAGE = 1.25;
+
+export class Sword extends Weapon {
+  constructor(
+    baseDamage: number,
+    baseDurability: number,
+    value: number,
+    weight: number
+  ) {
+    super("sword", baseDamage, baseDurability, value, weight);
+  }
+
+  polish(): void {
+    if (
```

Comment:
1. Please, use getter/setter for `damageModifier` and `baseDamage` (these properties should be private according to the UML diagram)
2. Wrong implementation - the maximum value of `damageModifier` should be less then 25% the of `baseDamage`.
3. Let's make `MODIFIER_CHANGE_RATE` static const of the Weapon class

Reply:
done